### DISCUSSION TITLE
Video how to?

### DISCUSSION THREAD

**Discussion by P49K** [GerryDazoo/Slinger#65] [General]
Do you think there will be a video how to for those of us that need the extra help getting slinger to work.
The project sounds exciting and I would like to continue using my slingbox/s after November.

Thank you for the work involved 

Ken

**Comment by GerryDazoo** [comment]
Sorry not in the cards, I'm working on some more detailed documentation. 
In the meantime, run it with all the defaults ( You'll need to set the slingbox type and password  in the config.ini ). There's an excellent chance it will just work if your slingbox is on the same network as the server.

**Reply by P49K** [reply]
I am assuming the documentation is for the slingbox.   I hope to be able to read it and then determine
if I can move forward trying it.

Thanks
Ken

**Reply by GerryDazoo** [reply]
No the documentation would be for my program. Just grab the code and run it out of the box after configuring your slingbox password. There a good chance it will just work.

**Comment by surveyor2** [comment]
I haven't tried the remote stuff yet but it was my first attempt tonight and I got the video/audio working on VLC fine. The only bit where I got confused was setting the config.ini video source to 2 for component on my M1 and it should have been 1. Perhaps the sample config.ini file can be edited a bit? Excellent program though.

James
